packages/vue-components/src/Searchbar.vue

```diff
@@ -253,7 +263,8 @@ export default {
 
 <style scoped>
     .form-control {
-        min-width: 8em;
+        min-width: 12.7em;
+        max-width: 25.4em;
```

Comment:
What's the rationale for the max-width? I think it could help to add a comment to explain in case someone would want to change it in the future :)